frontend/src/App.jsx

```diff
@@ -7,10 +7,10 @@ import {
   Route
 } from "react-router-dom";
 
-import { DASHBOARD, PROFILE, CREATE_RIDE, USER_RIDES } from './helper_assets/menu-paths';
+import { DASHBOARD, PROFILE, RIDE, USER_RIDES } from './helper_assets/menu-paths';
```

Comment:
this comment is for line `2`:

we don't `css` directory in frontend!. the app is crashing  coz of that line.